### DISCUSSION TITLE
GitHub Issues

### DISCUSSION THREAD

**Discussion by elyakoutiweb** [community/community#147705] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

How can I efficiently link GitHub Issues to a Project Board?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by xDweeb** [comment, marked as answer]
Go to the Issue, click the “Projects” sidebar, and select the desired project board. You can also automate this with project workflows to organize issues based on labels or milestones.
